backend/packages/Upgrade/src/api/services/ExperimentService.ts

```diff
@@ -154,7 +142,8 @@ export class ExperimentService {
 
     queryBuilder = queryBuilder.skip(skip).take(take);
```

Comment:
This is no longer needed as we have added limit and offset

Reply:
updated